### ISSUE TITLE
Links for changelog and blog post do not meet accessibility criterion

### ISSUE THREAD

**Issue by wojtekmaj** [nodejs/nodejs.org#7246]
### URL:

https://nodejs.org/en/download/package-manager

### Browser Name:

n/a

### Browser Version:

n/a

### Operating System:

n/a

### How to reproduce the issue:

On the page above, we have two links:

> Read the changelog for [this version](https://github.com/nodejs/node/releases/tag/v22.11.0)
> Read the blog post for [this version](https://nodejs.org/en/blog/release/v22.11.0)

This fails accessibility success criterion specified here:
https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html

It's basically as bad as "click here" link.

It would make more sense to write:

> Read the [changelog](https://github.com/nodejs/node/releases/tag/v22.11.0) for this version
> Read the [blog post](https://nodejs.org/en/blog/release/v22.11.0) for this version

instead.

**Comment by TenzDelek** [comment]
should i raise one pr for this or you are going for it? @wojtekmaj
